README.md

```diff
@@ -164,7 +164,7 @@ Sample request body:
 #### PUT
 
 Updates the status of a trip given the trip ID. The status is given in the body
-with the field `status`.
+with the field `status`. An optional `intermediateDestinationIndex` field can be passed in case this is a multi-waypoint trip and driver is moving between intermediate destinations.
```

Comment:
nit: "multi-waypoint" -> "multi-destination". Multi-waypoint trips can also refer to back-to-back trips. https://developers.google.com/maps/documentation/transportation-logistics/on-demand-rides-deliveries-solution/trip-order-progress/tasks/multi_waypoint

Reply:
Good point, updated